src/tools/http-tools.ts

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable eslint-comments/disable-enable-pair */
```

Comment:
The main branch now has an HTTP service, would be good to combine these approaches.